index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Blog</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-bar">
+            <div class="hamburger">
+                <span class="bars hamburger-bars"></span>
+                <span class="bars hamburger-bars"></span>
+                <span class="bars hamburger-bars"></span>
+            </div>
+            <ul class="menu">
+              <li><a href="#">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Speaker</a></li>
+              <li><a href="#">Sponsors</a></li>
+              <li><a href="#">News</a></li>
+              <li><a href="#">Register Now</a></li>
+           </ul>
+        </nav>
+    </header>
+    <section class="hero_section">
+
+        <div class="content1">
+         <h2>"Register Now"</h2>
+         <h1>UNLOCK THE </br> POWER OF DATA </br> SCIENCE WITH AI</h1>
+         <p>Welcome to 'Driven into Data Science</br> Using AI 2023',a premier event dedicated</br>
+            to exploring the intersection of data</br>science and artificial intelligence.Join us<br/>
+            for an immersive experience that will<br/>empower you with the knowledge and<br/>
+            skills needed to thrive in the data driven<br/> world. 
+         </p>
+
+        </div>
+         
+        <div class="venue">
+            <h2><strong>2023.10.15(THU) ~ 16(FRI)</strong></h2>
+            <h4>@ Field lane of Ghana Art Center Nabi and more</h4>
```

Comment:
This text `<h4> Field lane of Ghana. Art Center Nabi and more</h4>` should use the font family of `Nanum Gothic and font-size of 16px with color of rbg(51,51,51) ` according to the Figma design. Please fix this.